### ISSUE TITLE
validation loss -16.83, but performance is not better than model.h5.

### ISSUE THREAD

**Issue by LeeGyuHa** [breizhn/DTLN#33]
model.h5 : /pretrained_model/model.h5

As a result of performance comparison between my_model.h5 and model.h5, model.h5 is the best.
I looked at the reason.
The size of the weights(weight,bias) values of "my_model.h5" is small compared to the weights values of "model.h5"

figure1. weights result
![image](https://user-images.githubusercontent.com/35483975/109915945-16b9b380-7cf6-11eb-97c8-ca6c38669242.png)

figure2. signal result
![image](https://user-images.githubusercontent.com/35483975/109916604-41f0d280-7cf7-11eb-9e78-fed358fe5ac8.png)

The parameters are the same as the code. The dataset was DNS-Challenge2020. I used clean,noise.
Training data SNR range: -5 ~ 25 dB
Training data was created with the code provided by DNS-Challenge2020.
I set it to epoch 200, but because it is set to patient 10, it stopped at 96.
validation loss : -16.83

Is there a way to increase the weight value?
Is it a dataset issue?
Any advice would be appreciated.
Thank you.

**Comment by yemifeng** [comment]
Have you solved this problem? The same phenomena also happens in my experiment.

**Comment by zqlsnr** [comment]
hello, the parameters of your training model is 4003624？ 3990352？ Thanks！
